hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieWriteConfig.java

```diff
@@ -456,13 +456,6 @@ public class HoodieWriteConfig extends HoodieConfig {
       .withDocumentation("Whether to allow generation of empty commits, even if no data was written in the commit. "
           + "It's useful in cases where extra metadata needs to be published regardless e.g tracking source offsets when ingesting data");
 
-  public static final ConfigProperty<Boolean> ALLOW_OPERATION_METADATA_FIELD = ConfigProperty
-      .key("hoodie.allow.operation.metadata.field")
-      .defaultValue(false)
-      .sinceVersion("0.9.0")
-      .withDocumentation("Whether to include '_hoodie_operation' in the metadata fields. "
-          + "Once enabled, all the changes of a record are persisted to the delta log directly without merge");
```

Comment:
@XuQianJin-Stars  can you help me understand the main problem of the config staying in write config? Also, for user-facing config change, can we please file jira for tracking?